src/acceptance/java/io/codurance/pouch/AddResourceFeature.java

```diff
@@ -39,18 +39,17 @@ public void setUp() {
 
     @Test
     public void shouldAddOneResource() {
-        Map<String, String> resourceToAdd = new HashMap<>();
-        resourceToAdd.put("added", "2018-12-18 12:30:00.0");
-        resourceToAdd.put("title", "Stack Overflow - Where Developers Learn, Share, & Build Careers");
-        resourceToAdd.put("url", "https://stackoverflow.com");
+        Map<String, String> resourceToAdd = new HashMap<>(){{
+            put("title", "Stack Overflow - Where Developers Learn, Share, & Build Careers");
+            put("url", "https://stackoverflow.com");
+        }};
 
         given().contentType(JSON)
                 .body(resourceToAdd)
                 .when()
                 .post("/resources")
                 .peek()
                 .then()
-                .body("added", equalTo("2018-12-18 12:30:00.0"))
```

Comment:
Shall we assert that a timestamp is present, no matter what it is?

Reply:
I got a bit confused about this at first, as I had just removed the timestamp from the input following Neil's advice, but I see your point now. Well, I guess checking the presence and/or the format of the timestamp would improve the test. I will just have to find a suitable `Matcher` to assert on that.